### ISSUE TITLE
Uncaught (in promise) RangeError: Offset is outside the bounds of the DataView

### ISSUE THREAD

**Issue by yonghah** [flatgeobuf/flatgeobuf#329]
After I update to 3.27 from 3.24 in JS, I am often getting an error like below:

```
packedrtree.ts:179  Uncaught (in promise) RangeError: Offset is outside the bounds of the DataView
    at DataView.getBigUint64 (<anonymous>)
    at packedrtree.ts:179:1
    at streamSearch (packedrtree.ts:189:1)
    at streamSearch.next (<anonymous>)
    at HttpReader.selectBbox (http-reader.ts:139:1)
    at async deserializeFiltered (featurecollection.ts:131:1)
    at async loadFGB (loadFeaturesFromFGB.js:18:1)
    at async RoiControls.js:23:1

```
The JS part is nothing special and no change from 3.24

```
const results = [];
  for await (var fgb_src of fgb_srcs) {
    const iter = geojson.deserialize(
      fgb_src, bounds,
      (meta) => { console.log(meta); });
    for await (let f of iter) {
      results.push(f);
    }
    setCount(results.length);
  }
```
It seems that it is related to the recent change made in 
https://github.com/flatgeobuf/flatgeobuf/commit/cb12470a30441119472d24fb74b08b27b81aaea1

Are there any other getting the same error? It happens quite often. But I'm not able to reproduce precisely.

**Comment by bjornharrtell** [comment]
Thanks @yonghah for reporting this seems unexpected. I might consider rolling back but would really like to reproduce and understand who this could happen.

**Comment by michaelkirk** [comment]
Is the data something you can share? Would you be able to add some logging so you can see which bounding box query exhibits the failed behavior?

**Comment by yonghah** [comment]
@bjornharrtell @michaelkirk 

The dataset is
https://hd20-hlnjc-human-l2.s3.amazonaws.com/asset/sge/full_sdge.fgb

And the boundary is:
{maxX: 98245.28445015292, maxY: 1030.597977715265, minX: 98124.34087916819, minY: 900.6956236546853}

Both are in EPSG:3857.  This is the metadata from deserialize function:
```
{
    "geometryType": 1,
    "columns": [
        {
            "name": "gene_name",
            "type": 11,
            "title": null,
            "description": null,
            "width": 0,
            "precision": -1,
            "scale": -1,
            "nullable": true,
            "unique": false,
            "primary_key": false
        },
        {
            "name": "cnt_total",
            "type": 3,
            "title": null,
            "description": null,
            "width": 0,
            "precision": -1,
            "scale": -1,
            "nullable": true,
            "unique": false,
            "primary_key": false
        },
        {
            "name": "Gene",
            "type": 3,
            "title": null,
            "description": null,
            "width": 0,
            "precision": -1,
            "scale": -1,
            "nullable": true,
            "unique": false,
            "primary_key": false
        },
        {
            "name": "GeneFull",
            "type": 3,
            "title": null,
            "description": null,
            "width": 0,
            "precision": -1,
            "scale": -1,
            "nullable": true,
            "unique": false,
            "primary_key": false
        },
        {
            "name": "VelocytoSpliced",
            "type": 3,
            "title": null,
            "description": null,
            "width": 0,
            "precision": -1,
            "scale": -1,
            "nullable": true,
            "unique": false,
            "primary_key": false
        },
        {
            "name": "VelocytoUnspliced",
            "type": 3,
            "title": null,
            "description": null,
            "width": 0,
            "precision": -1,
            "scale": -1,
            "nullable": true,
            "unique": false,
            "primary_key": false
        },
        {
            "name": "VelocytoAmbiguous",
            "type": 3,
            "title": null,
            "description": null,
            "width": 0,
            "precision": -1,
            "scale": -1,
            "nullable": true,
            "unique": false,
            "primary_key": false
        }
    ],
    "envelope": null,
    "featuresCount": 13451255,
    "indexNodeSize": 16,
    "crs": {
        "org": "EPSG",
        "code": 3857,
        "name": "WGS 84 / Pseudo-Mercator",
        "description": null,
        "wkt": "PROJCRS[\"WGS 84 / Pseudo-Mercator\",BASEGEOGCRS[\"WGS 84\",ENSEMBLE[\"World Geodetic System 1984 ensemble\",MEMBER[\"World Geodetic System 1984 (Transit)\"],MEMBER[\"World Geodetic System 1984 (G730)\"],MEMBER[\"World Geodetic System 1984 (G873)\"],MEMBER[\"World Geodetic System 1984 (G1150)\"],MEMBER[\"World Geodetic System 1984 (G1674)\"],MEMBER[\"World Geodetic System 1984 (G1762)\"],MEMBER[\"World Geodetic System 1984 (G2139)\"],ELLIPSOID[\"WGS 84\",6378137,298.257223563,LENGTHUNIT[\"metre\",1]],ENSEMBLEACCURACY[2.0]],PRIMEM[\"Greenwich\",0,ANGLEUNIT[\"degree\",0.0174532925199433]],ID[\"EPSG\",4326]],CONVERSION[\"Popular Visualisation Pseudo-Mercator\",METHOD[\"Popular Visualisation Pseudo Mercator\",ID[\"EPSG\",1024]],PARAMETER[\"Latitude of natural origin\",0,ANGLEUNIT[\"degree\",0.0174532925199433],ID[\"EPSG\",8801]],PARAMETER[\"Longitude of natural origin\",0,ANGLEUNIT[\"degree\",0.0174532925199433],ID[\"EPSG\",8802]],PARAMETER[\"False easting\",0,LENGTHUNIT[\"metre\",1],ID[\"EPSG\",8806]],PARAMETER[\"False northing\",0,LENGTHUNIT[\"metre\",1],ID[\"EPSG\",8807]]],CS[Cartesian,2],AXIS[\"easting (X)\",east,ORDER[1],LENGTHUNIT[\"metre\",1]],AXIS[\"northing (Y)\",north,ORDER[2],LENGTHUNIT[\"metre\",1]],USAGE[SCOPE[\"Web mapping and visualisation.\"],AREA[\"World between 85.06°S and 85.06°N.\"],BBOX[-85.06,-180,85.06,180]],ID[\"EPSG\",3857]]",
        "code_string": null
    },
    "title": null,
    "description": null,
    "metadata": null
}
```

**Comment by bjornharrtell** [comment]
@yonghah this should hopefully be all fixed now, released as 3.27.1

**Comment by yonghah** [comment]
@bjornharrtell  It works well now. Thank you!
